content/en/docs/Installation/_index.md

```diff
@@ -380,6 +457,8 @@ The following flags can also be used to control the advanced behavior of Kyverno
 9. `autoUpdateWebhooks`: Set this flag to 'false' to disable auto-configuration of the webhook. Default is set to 'true'.
 10. `imageSignatureRepository`: specifies alternate repository for image signatures. Can be overridden per rule via `verifyImages.Repository`.
 11. `webhookRegistrationTimeout`: specifies the length of time Kyverno will try to register webhooks with the API server. Defaults to `120s`.
+12. `clientRateLimitQPS`: configure the maximum QPS to the control plane from Kyverno. Uses the client default if zero.
```

Comment:
Should we add some values here from a recent issue which mentioned some defaults that worked for clusters with high number of CRDs?

Reply:
https://github.com/kyverno/kyverno/issues/3440#issuecomment-1078439597

Reply:
Adding `20` and `50` to each of those as examples.